Project2015To2017.Migrate2017.Library/Transforms/FileTransformation.cs

```diff
@@ -47,6 +47,11 @@ public FileTransformation(ILogger logger = null)
 
 		public void Transform(Project definition)
 		{
+			if (definition.IsModernProject)
```

Comment:
It's better to add `ILegacyOnlyProjectTransformation` interface to `FileTransformation` instead of putting an `if` statement here. This way, user can override this by command-line switch.

Reply:
It's already `ILegacyOnlyProjectTransformation`, so the problem is actually more profound. I'm taking over the issue to investigate.